src/components/VAccordion.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <div class="accordion">
+    <div
+      class="accordion-header"
+      :class="{'accordion-header-active': visible}"
+      tabindex="0"
+      @click="open"
+      @keypress.enter.space="open"
+    >
```

Comment:
no real reason - it's just not my style to use them. do you want me to change it? this reminds me that i need to add in accessibility stuff

Reply:
using a button would get us `tabindex="0"`, `role="button"`, the keyboard handling for free, also other nice things that happen when a browser knows something is a button, like magic button focus style behavior. I think a button makes sense here.

I know this does mean w'll want to add styles to give this a more neutral default appearance. It would be nice to have a mixin for this, here's something we can start with: https://github.com/nypublicradio/nypr-pet-elections-client/blob/master/app/styles/mixins/_buttons.scss#L2

Reply:
hmm buttons can't be flex containers, so the icon is not aligned center/right anymore... and we want the entire header including the icon to be clickable. not sure it will work

Reply:
hmm i didn't know about this. it looks like this was fixed in chrome, uhhh last week? https://www.chromestatus.com/feature/5213452823953408 but it wont' be fixed in safari.
I hate to say it but can we wrap the button contents in a div and make that the flex container? I'd still like to get the "magic" button focus behavior (as mentioned here https://github.com/WICG/focus-visible/blob/master/explainer.md#rationale (pretty sure it's not just chrome)) so we can have focus state for keyboard that doesn't linger on mouse/touch interactions, without installing this plugin and setting custom focus rules right now.  If that sounds too crazy then let's at least include role=button on the div

Reply:
it seems like the ideal accessibility recommendation is to also wrap the whole header in an <h#> element so it shows up in lists of page headers, unfortunately as a standalone component we wont necessarily know what level it's going to be at... Maybe worry about this part later. https://www.w3.org/TR/wai-aria-practices-1.1/#accordion

Reply:
Or, do we need this to be a flex container? i feel like button with a right aligned icon is a thing that could be done before flexbox.... maybe that was only with an absolute positioning hack though, in which case nevermind

Reply:
Actually, yeah we can just go back to the div and add the button role and if anyone notices any problems with lingering focus outlines we can always fix it later as a bug

Reply:
ok sounds good :) lol!

Reply:
I left the header open to interpretation... it should be an h* but I'd rather not force which one people use. Unless you think we should?

Reply:
yeah i don't think theres a non-convoluted way to make it flexible. Let's leave it off for now and we can always come back to it later when we get a feel for how it's actually being used in pages